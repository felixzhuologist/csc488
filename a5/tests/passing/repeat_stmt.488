{
    var i : Integer
    i := 0
    repeat {
        write i, newline
        i := i + 1
     } until i = 2
    
    write "after", newline
}