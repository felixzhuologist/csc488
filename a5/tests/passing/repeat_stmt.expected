0
1
after
